python/ray/tune/sync_client.py

```diff
@@ -0,0 +1,141 @@
+from __future__ import absolute_import
```

Comment:
this is just a move right?

Reply:
yeah idt i changed anything

Reply:
also moved `log_sync.py` contents into `syncer.py` but there are some minor changes there (eg having `NodeSyncer` extend `LogSyncer` instead of using mixin